src/main/java/org/springframework/samples/petclinic/owner/PetTypeFormatter.java

```diff
@@ -51,12 +51,8 @@ public String print(PetType petType, Locale locale) {
 	@Override
 	public PetType parse(String text, Locale locale) throws ParseException {
 		Collection<PetType> findPetTypes = this.owners.findPetTypes();
-		for (PetType type : findPetTypes) {
-			if (type.getName().equals(text)) {
-				return type;
-			}
-		}
-		throw new ParseException("type not found: " + text, 0);
+		return findPetTypes.stream().filter(petType -> petType.getName().equals(text)).findFirst()
```

Comment:
I'm not sure this is an improvement really.

Reply:
It is a spring sample project, and it may not make sense because there is no performance improvement.

However, I thought it would be better for people who come to see the Spring sample project to see the Java 8 stream API than forEach.

I'm curious what you think.

---
It might be awkward because the sentence is being translated.
Please excuse me. 

I think I'm very lucky to talk to a good developer.
Thank you!

Reply:
I don’t think the purpose of this sample project is to promote the use of Java 8 APIs. It’s better to use whatever makes sense imo, and in this case the old syntax is shorter and easier to understand.

Reply:
understand! 
commit 3720069 will roll back.